tests/TestCase/Database/Schema/PostgresSchemaTest.php

```diff
@@ -217,11 +217,13 @@ public function testConvertColumn($type, $expected)
             'default' => 'Default value',
             'comment' => 'Comment section',
             'char_length' => null,
+            'collation_name' => 'Collate information',
         ];
         $expected += [
             'null' => true,
             'default' => 'Default value',
             'comment' => 'Comment section',
+            'collate' => 'Collate information',
```

Comment:
Apparently, I copied-and-pasted from [MysqlSchemaTest.php](https://github.com/cakephp/cakephp/blob/a6f8945561c7570fbbe82e73c5ba378846059a81/tests/TestCase/Database/Schema/MysqlSchemaTest.php#L190).